source/policies/decision-making.rst

```diff
@@ -0,0 +1,60 @@
+Decision Making Policy
+======================
+
+All decision making processes described below are written to ensure
+transparency.
+
+Decisions
+---------
+
+When a decision is made this is to denoted in an :code:`rst` file in the
+:code:`ukpa-internaldocs/source/decision/` subdirectory.
+
+Examples of these include:
+
+- :doc:`../decisions/2020-02-15-add-policy-on-minors`
+- :doc:`../decisions/2022-11-19-treasurer-and-chair`
+
+Decisions can lead to a policy of the UKPA.
+
+Agreement of decisions
+**********************
+
+For a decision to be agreed the following has to occur:
+
+1. A pull request on the `ukpa internaldocs github repository
+   <https://github.com/UKPythonAssociation/ukpa-internaldocs>`_ must be opened
+   with the modifications recorded in a :code:`source/decisions/<file.rst>`.
+2. This pull request should be labeled with either :code:`Trustee Meeting
+   Decision` or :code:`Resolution Decision` as appropriate.
```

Comment:
Should these be defined (or linked to, if it's defined somewhere else already)?  Is a Resolution Decision just any decision that doesn't happen at a Trustee meeting, or is there something more to it?

Reply:
Section 17 of our constitution defines two ways in which trustees may make decisions - at a meeting or by resolution  - and they have different requirements for us to meet in order for them to be valid.